### DISCUSSION TITLE
SvelteKit Page Endpoint get URL Params

### DISCUSSION THREAD

**Discussion by ghost** [sveltejs/kit#5214] [General]
Exploring the powerful Page Endpoint:

.ts:

```
import type {RequestHandler} from "@sveltejs/kit";
const url="https://example.com?page="+page;
const res=await fetch(url);
const data=await res.json();
export const get:RequestHandler=()=>{
return{
body:{
data
}
}
}
```

Can I get the "page" parameter from the current URL [ex example2.com?p=10] in the .ts and pass to above? I know its possible in .svelte

**Comment by ghost** [comment]
Update: I can get the event.url within the get:RequestHandler but would like to get it prior to setting the const url above

**Comment by Aqirito** [comment]
any update here?
